### PR TITLE
Login to pyrog with login/password instead of using the token

### LINKED ISSUE #31: Use login mutation with identifiers instead of pyrog token

### PR DESCRIPTION
**Pull request by simonvadee**
Fix #31

### CODE REVIEW

File: pagai/services/pyrog/pyrog.py [hunk 1]
```diff
@@ -37,44 +46,67 @@
...
-        )
+        return headers
 
-    try:
-        response = requests.post(
-            PYROG_URL, headers=get_headers(), json={"query": graphql_query, "variables": variables}
-        )
-    except requests.exceptions.ConnectionError:
-        raise OperationOutcome("Could not connect to the Pyrog service")
+    def run_graphql_query(self, graphql_query, variables=None, auth=True):
```
Comment (Jasopaum) [inline pagai/services/pyrog/pyrog.py:64]: Can you add just one small line to explain the auth argument?
Also, I think something like `auth_needed` or `protected` or I don't know what would make it more understandable

File: .env [hunk 2]
```diff
@@ -7,5 +7,7 @@ PG_DB_SCHEMA=
 STR_ID_MAX_UNIQUE_VALUES=40
 INCLUDE_THRESHOLD=0.5
 MIN_SCORE=0.1
-PYROG_URL=http://localhost:4000
-PYROG_TOKEN=<token>
\ No newline at end of file
+PYROG_URL=http://localhost:1000
+#PYROG_URL=https://pyrog.staging.arkhn.com/pyrog/
```
Comment (Jasopaum) [inline .env:11]: Do we want to keep this line?

Comment (simonvadee) [reply .env:11]: yes, I think it's fine

File: pagai/services/pyrog/pyrog.py (hunk 1 above)
Comment (simonvadee) [reply pagai/services/pyrog/pyrog.py]: that's better indeed! done.

**Pull request merged**